### PR TITLE
Remove duplicate cherrypy logs in the central services

### LINKED ISSUE #8111: Duplicate reqmgr log records
Sorry, my bad that I haven't seen this one before... apparently this module
https://github.com/dmwm/WMCore/blob/master/src/python/WMCore/Services/ReqMgr/ReqMgr.py

is writing its log records twice to the reqmgr log file, e.g.:
```
[30/Aug/2017:11:46:35]  INFO: Assign request amaltaro_ReDigi_cmsR ...
then
INFO:cherrypy.error:[30/Aug/2017:11:46:35]  INFO: Assign request amaltaro_ReDigi_cmsR...
```

which means - I think - that logger object has two handlers. I'm not sure I know how to fix it without breaking other stuff :-)

### PR DESCRIPTION
**Pull request by amaltaro**
Fixes #8111 and #8644

The actual change to stop the duplicate log records was just with the `log.screen=False` line, which disables writing log records to stdout. This way we keep only the cherrypy access and error logs (with a format that ain't great, but I couldn't fix it yet). Notice that everything that is not an HTTP request (access) is logged with the error log manager, see documentation in:
https://tools.ietf.org/doc/python-cherrypy3/api/cherrypy._cplogging-module.html

Second change was to make ReqMgr2 slightly more verbose, recording original user parameters, priority and status transitions.
Seangchan, as we discussed, logging for priority change of requests not assigned yet are ugly, since the web UI posts to ReqMgr server the whole dictionary built for the client's browser.

### CODE REVIEW

**Comment by amaltaro** [comment]
Another issue that I don't think we can fix are those print statements before the logging is setup, given that we disable the screen log, I don't see them in my tests. :(

**Comment by amaltaro** [comment]
@ticoann I still want to do some pylint cleanup, but please review. And if you're concerned about the logging format or something else, we can discuss tomorrow. I think we can wait until Monday for the next testbed

File: src/python/WMCore/REST/Main.py [hunk 1]
```diff
@@ -85,29 +85,28 @@ def access(self):
...
-                   and getattr(request.rfile.rfile, "bytes_read", None)
-                   and request.rfile.rfile.bytes_read) or "-",
-             'b': nout or "-",
-             'T': delta_time,
-             'AS': inheaders.get("CMS-Auth-Status", "-"),
-             'AU': inheaders.get("CMS-Auth-Cert", inheaders.get("CMS-Auth-Host", "")),
-             'AC': getattr(request.cookie.get("cms-auth", None), "value", ""),
-             'f': inheaders.get("Referer", ""),
-             'a': inheaders.get("User-Agent", "") })
+        msg = ('%(t)s %(H)s %(h)s "%(r)s" %(s)s'
```
Comment (ticoann) [inline src/python/WMCore/REST/Main.py]: where does this msg is used now? I don't see any place here. Am I missing something?

**Comment by amaltaro** [comment]
@ticoann I just pushed another commit moving the log.screen thing down to the moment we run the server. That way we can get all the initial setup logging out (including a dump of the service config) and still get rid of duplicate logs.
If you think it makes more sense, I can squash those commits.

**Comment by ticoann** [comment]
Thanks Alan, patch looks good to me.

**Pull request merged**